tests/unittest/llmapi/test_llm.py

```diff
@@ -534,6 +534,41 @@ async def main():
     test_non_streaming_usage_wait()
 
 
+@pytest.mark.parametrize("chunked", [True, False])
+@pytest.mark.part0
+def test_llm_generate_async_with_stream_interval(chunked):
+    model_path = f"{llm_models_root()}/nvfp4-quantized/Meta-Llama-3.1-8B"
+    build_config = BuildConfig()
+    build_config.max_num_tokens = 256
+    with LLM_torch(model_path,
+                   stream_interval=4,
+                   build_config=build_config,
+                   enable_chunked_prefill=chunked) as llm:
+        sampling_params = SamplingParams(max_tokens=13,
+                                         ignore_eos=True,
+                                         detokenize=False)
+        step = 0
+        last_step_len = 0
+        prompt = "The capital of France is "
+        if chunked:
+            prompt = prompt * build_config.max_num_tokens
+        for output in llm.generate_async(prompt,
+                                         sampling_params=sampling_params,
+                                         streaming=True):
+            current_step_len = len(output.outputs[0].token_ids)
+            # The output lens of each step need to be [1, 3, 4, 4, 1]
```

Comment:
Can I understand this list as below?
- For `1`, the context phase generates only one token and should return immediately for TTFT
- For `3`, since `(1+3) % 4 == 0`
- For `4`, since `(1+3+4) % 4 == 0`
- For `4`, same reason as above
- For `1`, it reatch max_tokens and stop